internal/cmd/imagerunner/artifacts.go

```diff
@@ -80,7 +80,7 @@ func downloadCommand() *cobra.Command {
 	var out string
 
 	cmd := &cobra.Command{
-		Use:   "download",
+		Use:   "download <runID> <file-pattern>",
```

Comment:
In `saucectl artifacts download` we call it `"download <jobID> <filename>"`. Should keep it consistent with regards to filename or file-pattern (I prefer the latter, as you did, since we do indeed support patterns, which we do mention in the _short_ description of the command, but is not immediately obvious by looking at the params. 

TLDR; for consistency, it'd be great if you could update other instances of `<filename>` to `<file-pattern>` where applicable.

Reply:
It's updated ! `upload` does not supports it, so it's kept as it was before.